src/Microsoft.Extensions.Caching.Memory/MemoryCacheOptions.cs

```diff
@@ -9,10 +9,41 @@ namespace Microsoft.Extensions.Caching.Memory
 {
     public class MemoryCacheOptions : IOptions<MemoryCacheOptions>
     {
+        private long? _sizeLimit;
+        private double _compactionPercentage = 0.05;
+
         public ISystemClock Clock { get; set; }
 
         public TimeSpan ExpirationScanFrequency { get; set; } = TimeSpan.FromMinutes(1);
 
+        public long? SizeLimit
+        {
+            get => _sizeLimit;
+            set
+            {
+                if (value <= 0)
```

Comment:
allow 0 for test scenarios. e.g. temporarily reduce the cache limit to 0 to debug an issue masked (or caused) by caching.